interview-questions.md

```diff
@@ -5,57 +5,115 @@ Answer the following questions. First, without external resources. Challenge you
 1. Why would you use the method super()?
 
   Your answer:
+  You would use the method super() in react if a state is needed for a component. You also use it inside a constructor for a class.
 
   Researched answer:
+  The super keyword refers to the parent class.
+
+  It is used to call the constructor of the parent class and to access the parent's properties and methods.
+
+  You use it in a constructor when the constructor is extending another class and adding more properties & you want the extended class to include all the properties of the parent class.
+
+  The parent class doesn't need super because it is at the top of the prototypal inheritance chain.
 
 
 
 2. What is a virtual DOM?
 
-  Your answer:
+  Your answer: it is the space between the code and what renders that is making decisions about how things will render on screen depending on user behavior or changes.
```

Comment:
Totally. I also think it would be wise to break down what DOM stands for as well.